{"errorMessage": "An error occurred (UnauthorizedOperation) when calling the DescribeInstances operation: You are not authorized to perform this operation. User: arn:aws:sts::266735808339:assumed-role/lambda-piped-2-MyLambdaFunctionRole-9l5zRkvZg1r7/lambda-piped-fn is not authorized to perform: ec2:DescribeInstances because no identity-based policy allows the ec2:DescribeInstances action", "errorType": "ClientError", "requestId": "f999fca0-f3c9-43a7-bcda-c19474fde3d9", "stackTrace": ["  File \"/var/task/main.py\", line 6, in lambda_handler\n    response = ec2_client.describe_instances()\n", "  File \"/var/lang/lib/python3.12/site-packages/botocore/client.py\", line 569, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n", "  File \"/var/lang/lib/python3.12/site-packages/botocore/client.py\", line 1023, in _make_api_call\n    raise error_class(parsed_response, operation_name)\n"]}